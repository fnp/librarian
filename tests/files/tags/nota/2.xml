<!-- lub czasem na końcu utworu: -->

<naglowek_czesc>III</naglowek_czesc>

<strofa>
Każdego z takich, jak ty, świat nie może/
Od razu przyjąć na spokojne łoże,/
I nie przyjmował nigdy, jak wiek wiekiem./
Bo glina w glinę wtapia się bez przerwy,/
Gdy sprzeczne ciała zbija się aż ćwiekiem/
Później... lub pierwéj...
</strofa>

<nota><akap>Pisałem w Paryżu 1856 w styczniu.</akap></nota>
