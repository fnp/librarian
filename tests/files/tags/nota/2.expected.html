<div id="book-text">
<div id="toc">
<h2>Spis treści</h2>
<ol>
<a href="#s1" class="anchor">III</a><li>
</ol>
</div>
<h2 class="wl" id="s1">III</h2>
<div class="stanza" id="i1">
<a href="#f1" class="wl-num">1</a><div class="wl verse" id="f1">Każdego z takich, jak ty, świat nie może</div>
<div class="wl verse" id="f2">Od razu przyjąć na spokojne łoże,</div>
<div class="wl verse" id="f3">I nie przyjmował nigdy, jak wiek wiekiem.</div>
<div class="wl verse" id="f4">Bo glina w glinę wtapia się bez przerwy,</div>
<a href="#f5" class="wl-num">5</a><div class="wl verse" id="f5">Gdy sprzeczne ciała zbija się aż ćwiekiem</div>
<div class="wl verse" id="f6">Później… lub pierwéj…</div>
</div>
<div class="note"><p class="wl paragraph" id="i2">Pisałem w Paryżu 1856 w styczniu.</p></div>
</div>
